airflow/providers/celery/CHANGELOG.rst

```diff
@@ -27,6 +27,19 @@
 Changelog
 ---------
 
+3.3.2
+.....
+
+Features
+~~~~~~~~
+
+* ``Add missing re2 dependency to cncf.kubernetes and celery providers (#33237)``
```

Comment:
in Kubernetes provider we consider the change as Misc :thinking: 
IMO it should be Bug fix or Misc